msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/TestCase1561087.java

```diff
@@ -0,0 +1,56 @@
+package com.microsoft.identity.client.msal.automationapp.testpass.broker;
+
+import com.microsoft.identity.client.msal.automationapp.R;
+import com.microsoft.identity.client.ui.automation.annotations.SupportedBrokers;
+import com.microsoft.identity.client.ui.automation.broker.BrokerHost;
+import com.microsoft.identity.labapi.utilities.client.LabQuery;
+import com.microsoft.identity.labapi.utilities.constants.AzureEnvironment;
+import com.microsoft.identity.labapi.utilities.constants.TempUserType;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+// Flight settings
+// https://identitydivision.visualstudio.com/Engineering/_workitems/edit/1561087
+@SupportedBrokers(brokers = BrokerHost.class)
+public class TestCase1561087  extends  AbstractMsalBrokerTest {
+    @Test
+    public void test_1561087() throws Throwable {
+        // Set flights and get to check if the flight information is returned
+        final String flightsJson =  "{\"SetFlightsTest\":\"true\"}";
+        mBroker.setFlights(flightsJson);
+        Assert.assertEquals(mBroker.getFlights(), flightsJson);
```

Comment:
The order is that first param is supposed to be expected and the second one is actual value

```suggestion
        Assert.assertEquals(flightsJson, mBroker.getFlights());
```

Reply:
The above mentioned feedback applies here and everywhere